app/views/site/_study_settings_viz.html.erb

```diff
@@ -40,6 +40,12 @@
           <%= opts.text_field :expression_label, value: @study.default_options[:expression_label], placeholder: @study.default_expression_label, class: 'form-control' %>
         </div>
       </div>
+      <div class="form-group row">
```

Comment:
Blocker: One thing I noted in this form that is not a result of a change you've made, but rather a lurking bug - if you have a study w/o clustering data that does have gene lists, we cannot render this form.  We may need to abstract out these various sections into partials and render them iteratively depending on what kind of data the study has.

Reply:
Good catch -- will address